Berksfile

```diff
@@ -4,11 +4,11 @@ metadata
 
 # how come tests do not pick up these from metadata.rb? :/
 cookbook 'java', '>= 1.50'
-cookbook 'ulimit'
-cookbook 'apt'
-cookbook 'yum'
-cookbook 'ark'
-cookbook 'systemd'
-cookbook 'chef_handler', '~> 3.0.2'
-
-cookbook 'build-essential'
+cookbook 'ulimit', '>= 0.0.0'
```

Comment:
Do we have to mention > 0.0 version? without any version means it will take the latest version greater than 0.0

Reply:
No..I have removed it